frontend/src/components/LevelMissionInfoBanner/LevelMissionInfoBanner.tsx

```diff
@@ -25,6 +32,11 @@ function LevelMissionInfoBanner({
 				</span>
 				Mission Info
 			</ThemedButton>
+			{isLevelComplete && isLevel3 && (
+				<ThemedButton onClick={openLevelsCompleteOverlay}>
+					Congratulations
```

Comment:
Maybe a `!` would be nice? As in, "Congratulations!"

Reply:
sounds good